### DISCUSSION TITLE
Linearized Actuator

### DISCUSSION THREAD

**Discussion by gyger** [lneuhaus/pyrpl#538] [Q&A]
Thanks a lot for the nice tool, we were wondering if there is a possibility to create a linearized Actuator inside the tool (Control Valve Linearization). In our case we are thinking about using a VOA which has a somehow nonlinear control input.

**Comment by michaelcroquette** [comment]
Could you describe a bit more your problem ? Do you want to lock your system ? What is the transfer function and the error signal ?
